composer/models/resnet_cifar10/model.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 MosaicML. All Rights Reserved.
+
+"""ResNet models for CIFAR10 extending :class:`.ComposerClassifier`."""
+
+from typing import List, Optional
+
+from composer.models.base import ComposerClassifier
+from composer.models.model_hparams import Initializer
+from composer.models.resnet_cifar10.resnets import CIFARResNet, ResNet9
+
+__all__ = ["CIFAR10ResNet"]
+
+
+class CIFAR10ResNet(ComposerClassifier):
+    """ResNet models for CIFAR10 extending :class:`.ComposerClassifier`.
+
+    From `Deep Residual Learning for Image Recognition <https://arxiv.org/abs/1512.03385>`_ (He et al, 2015).
+
+    Args:
+        model_name (str): ``"cifar_resnet_9"``, ``"cifar_resnet_20"``, or ``"cifar_resnet_56"``.
```

Comment:
Given that ResNet9 is not a canonical ResNet, I might add a disclaimer to that effect and/or link directly to the ResNet9 documentation.